src/app/sell-item-page/sell-item-page.component.html

```diff
@@ -0,0 +1,39 @@
+<form>
+  <div class="form-group">
+    <label for="itemName">Item Name</label>
+    <input type="text" class="form-control" id="itemName" [(ngModel)]="itemName" name="Item Name" required>
+  </div>
+  <div class="form-group">
+    <label for="sellerID">Seller ID</label>
+    <input type="text" class="form-control" id="sellerID" [(ngModel)]="sellerID" name="Seller ID" required>
+  </div>
+  <div class="form-group">
+    <label for="conditionTypeID">Condition Type ID</label>
+    <input type="text" class="form-control" id="conditionTypeID" [(ngModel)]="conditionTypeID" name="Condition Type ID" required>
```

Comment:
Can we make this and also the categoryType dropdown selections, by using `<select/>` and `<options/>`?

Reply:
Yes, but I don't think that's necessary for the init. I think that will be done for full functionality. The init of this page is mostly so it is easier to add items to the database for testing. The next issue I will make and work on is making the add item page more functional.